ceph-dev-new-build/build/build_mingw

```diff
@@ -1,7 +1,25 @@
 #!/bin/bash
 set -ex
 
+# We need Ubuntu Focal to cross-compile Ceph for Windows.
+# Previous versions provide broken Mingw packages.
+# "DIST" will be set to "windows", so we're currently overriding it with
+# a hardcoded value.
+tmp_pbuild_script=$(mktemp /tmp/build_mingw_pbuild.XXXXXX)
+cat << EOF > $tmp_pbuild_script
```

Comment:
Nit: does the script need to be marked executable via `chmod +x` or does `pbuilder execute` not care?

Reply:
I think pbuilder handles that by itself, it worked on my env. I guess we can set it explicitly though.